core/src/main/java/google/registry/flows/domain/DomainPricingLogic.java

```diff
@@ -122,6 +128,64 @@ FeesAndCredits getRenewPrice(Registry registry, String domainName, DateTime date
             .build());
   }
 
+  /** Returns a new renewal cost for the pricer. */
+  @SuppressWarnings("unused")
```

Comment:
Once this is being used you shouldn't need this (also technically we already no longer need it on the existing `getRenewPrice()`, as that method is in fact used in 2 places, so its presence is already confusing).